### PR TITLE
Set up urls

### LINKED ISSUE #87: Set up URLs
__Description:__
We need to serve the landing at `clubhub.xyz` and the sites at `*.uvic.club`.

__Breakdown:__
- [x] Change main landing
- [x] Switch to `edit` from `editor`
- [x] Set up nginx for `*.uvic.club`

__Time Estimation:__
5 hrs.

### PR DESCRIPTION
**Pull request by hwkr**
In this branch:

* Sets up the main app site as `http://tryclubhub.com`
* Sets up the club sites as `*.uvic.club`
* Puts the editor page at `*.uvic.club/edit`
* Fixes buttons related to this
* Adds production and server configurations

Resolves #87.

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -61,15 +63,30 @@ export default class App extends PureComponent{
...
 
-  static propTypes = {
-    params: PropTypes.shape ({
-      siteId: PropTypes.string
-    })
+  getRouter = () => {
+    let names = window.location.host.split('.')
+    names = names[0] === ('www') ? names.slice(1) : names // trim www
+
+    if (Config.subhosts.indexOf(names.slice(1).join('.')) >= 0)
```
Comment (okjuan) [inline client/components/App.jsx:70]: I still think we shouldn't exclude curly braces.

File: client/components/App.jsx [hunk 2]
```diff
@@ -2,6 +2,8 @@ import React, {PureComponent, PropTypes} from 'react'
 import {Router, Route, browserHistory} from 'react-router'
 import Async from 'react-promise'
 
+import Config from 'Config'
```
Comment (okjuan) [inline client/components/App.jsx:5]: It's great that you set up the config!

File: client/components/App.jsx [hunk 3]
```diff
@@ -61,15 +63,30 @@ export default class App extends PureComponent{
...
+      // Sub host (club site)
+      return (
+        <Router history={browserHistory}>
+          <Route siteId={names[0]} component={SiteContainer} path="/" />
+          <Route siteId={names[0]} component={EditorContainer} path="/edit"/>
+        </Router>
+      )
+    else
+      // Main host (clubhub site)
+      return (
```
Comment (okjuan) [inline client/components/App.jsx:80]: For the sake of making the code easy to reason about -- especially if we add more routes -- I think we should have a single return statement.

File: server/main.js [hunk 4]
```diff
@@ -98,18 +98,25 @@ app.post('/api/site/*', function(req, res){
...
+app.post('/api/newsite/*', function (req, res) {
   let url = req.path.substring(req.path.lastIndexOf('/') + 1)
   let siteName = req.body.siteName
-  if (siteName)
+  if (siteName) {
     db.createNewSite(url, siteName, function (json) {
       res.json(json)
-    })
-  else
+    }) } else {
```
Comment (okjuan) [inline server/main.js]: small newline/indentation fix

**Review by okjuan — DISMISSED** [review]
Great job! I left a couple comments requesting changes, mainly regarding code style. Let me know what you think and then let's get this thing merged!

**Comment by hwkr** [comment]
If you can check out the code and make any style tweaks you want please do (make sure it lints okay). In terms of the single exit stuff I always think that's overrated but whatever works.

**Pull request merged**